src/detect-threshold.c

```diff
@@ -372,7 +379,41 @@ static int ThresholdTestParse05(void)
     return 0;
 }
 
+/**
+ * \test ThresholdTestParse06 is a test for thresholding by_both
+ *
+ *  \retval 1 on succces
+ *  \retval 0 on failure
+ */
+static int ThresholdTestParse06(void)
```

Comment:
it would be nice to switch the new tests to the FAIL/PASS API. See for example DetectUdphdrParseTest01

Reply:
Done in the v3 PR.